JOSE_paper/paper.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Research Software Engineering Skills'
+tags:
+  - Software Engineering
+  - Python
+  - Research Software
+authors:
+  - name: Jack Atkinson
+    orcid: 0000-0001-5001-4812
+    affiliation: "1"
+    corresponding: true
+  - name: Amy Pike
+    affiliation: "1"
+    orcid: 0009-0007-7278-4122
+  - name: Marion Weinzierl
+    affiliation: "1"
+    orcid: 0000-0002-2302-5476
+affiliations:
+ - name: Institute of Computing for Climate Science, University of Cambridge, UK
+   index: 1
+date: 29 July 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Research across many domains increasingly relies on software for simulation,
+data processing, analysis, and visualisation.
+This software is often written by those without formal training in software engineering
+leading to ineficciencies, bugs, and poor reproducibility/reuseability.
+
+The course described in this paper is intended to raise awareness of the importance of software
```

Comment:
I think it might be a good idea to switch these first two paragraphs around so that someone scanning the paper sees what's in it straight away.

Reply:
I am ok with leaving the order as it is